authors/apps/articles/serializers.py

```diff
@@ -0,0 +1,43 @@
+from rest_framework import serializers
+from .models import ArticlesModel
+
+
+class ArticlesSerializers(serializers.ModelSerializer):
+
+    title = serializers.CharField(
+        required=True,
+        max_length=128,
+        error_messages={
+            'required': 'Title is required',
+            'max_length': 'Title cannot be more than 128'
+        }
+    )
+    description = serializers.CharField(
+        required=False,
+        max_length=250,
+        error_messages={
+            'max_length': 'Description should not be more than 250'
+        }
+    )
+    body = serializers.CharField(
+        required=True,
+        error_messages={
+            'required': 'Body is required'
+        }
+    )
+
+    image_url = serializers.URLField(
+        required=False
+    )
+
+    class Meta:
+        model = ArticlesModel
+        fields = (
+            'title',
+            'description',
+            'body',
+            'slug',
+            'image_url',
+            'author',
+            'createdAt',
+            'updatedAt')
```

Comment:
For legibility, @FawazFarid, had recommended we have this as:
```python
    fields = (
        'title',
        'description',
        'body',
        'slug',
        'image_url',
        'author',
        'createdAt',
        'updatedAt'
    )
```
Note the closing bracket.